### ISSUE TITLE
[APPLY] apply for batch convert from DOC to DOCX | 申请doc 转 docx 批量转换功能

### ISSUE THREAD

**Issue by geekyouth** [jgm/pandoc#4875]
[APPLY] apply for batch convert from DOC to DOCX | 申请doc 转 docx 批量转换功能
![image](https://user-images.githubusercontent.com/12899262/45136177-83ab6b80-b1d5-11e8-8f72-9d4be65a3f22.png)

thanks

**Comment by agusmba** [comment]
The error you received is correct:
```
Pandoc can convert from DOCX, but not from DOC.
```

**Comment by mb21** [comment]
If you're asking for a `.doc`-reader, I think this is out of scope, since we already support docx and .doc is a hard to parse legacy binary format.

For questions, please ask on the pandoc-discuss mailing list.
